pixi.toml

```diff
@@ -23,15 +23,14 @@ pandas = "*"
 stackstac = "*"
 geopandas = "*"
 rioxarray = "*"
-rasterio = "1.3.*"
 networkx = "3.3"
 matplotlib = "*"
 typing-extensions = ">=4.12.0rc1"
 sqlite = "*"
+solara = "*"
 pip = "*"
 
+
 [target.linux-64.pypi-dependencies]
 mesa = { version = "==3.1.1" }
 mesa-geo = { version = "==0.9.0" }
```

Comment:
It would be great if we could get the needed versions of mesa-geo up on
conda-forge! The source of the proj problem seems to be mixing pip and conda
deps. Rasterio >=1.4b1 is required by mesa-geo 0.9:

https://github.com/projectmesa/mesa-geo/blob/bb1cfca2d799d4d5012d45578cc99dbb1c61af87/pyproject.toml#L46

And we specified a conda dependency of rasterio 1.3.*.